rachelnavarro/step/portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,15 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function drawCardOfTheDay() {
+  const deck =
+      ['The Fool', 'The Magician', 'The High Priestess', 'The Empress'];
+
+  // Draw a random card.
+  const card = deck[Math.floor(Math.random() * deck.length)];
```

Comment:
Wow js doesn't have a Math.random(min, max) sorta thing. Sad.

Reply:
I know right! That would be super helpful